.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "BreakpadSupport/breakpad-support/src/main/cpp/google-breakpad"]
 	path = BreakpadSupport/breakpad-support/src/main/cpp/google-breakpad
 	url = https://github.com/microsoft/appcenter-sdk-android-breakpad.git
+[submodule "Modules/Newtonsoft.Json-for-Unity"]
+	path = Modules/Newtonsoft.Json-for-Unity
+	url = https://github.com/jilleJr/Newtonsoft.Json-for-Unity.git
```

Comment:
We'll need to add an entry in the ThirdPartyNotices.txt for this one as it's shipped in the SDK.